src/CommandingOfficers/Commander.java

```diff
@@ -54,7 +54,7 @@
   public Color myColor;
   public static final int DEFAULTSTARTINGMONEY = 0;
   public static final int CHARGERATIO_FUNDS = 9000; // quantity of funds damage to equal 1 unit of power charge
-  public static final int CHARGERATIO_HP = 90; // quantity of HP damage dealt to equal 1 unit of power charge
+  public static final int CHARGERATIO_HP = 100; // Funds value of 1 HP damage dealt, for the purpose of power charge
```

Comment:
I don't think this comment makes sense. Funds value per 1HP varies by unit type.

Reply:
Actually, it makes perfect sense. It's putting a funds value to the HP damage you deal on your turn.

Reply:
"for the purpose of power charge" could be clarified, though

Reply:
Wait, are you still converting HP to funds, instead of to power? I don't think the math is working out the same was as it did before - we are now multiplying HP dealt by 100 instead of dividing by 90.